src/zope/app/dependable/dependency.py

```diff
@@ -16,6 +16,8 @@
 on an object having dependencies. It raises an exception if it's the
 case.
 """
+from __future__ import print_function, absolute_import, division
+
```

Comment:
I don't see any print statements or divisions in this module, and I'd be very much surprised if any of the `from zope.x.y.z` imports could be affected by `absolute_import`.